dss/operations/util.py

```diff
@@ -39,3 +44,137 @@ def map_bucket_results(func: typing.Callable, handle: BlobStore, bucket: str, ba
 def map_bucket(*args, **kwargs):
     for _ in map_bucket_results(*args, **kwargs):
         pass
+
+def set_cloud_env_var(
```

Comment:
The usage of `set_cloud_env_var `, and of `unset_cloud_env_var`, is difficult to understand, and appears restricted to `dss/operations/params.py`. Please refactor to move them into `dss/operations/params.py`, and remove their associated wrapper methods.

Reply:
Note that the wrapper methods were suggested by @DailyDreaming in [this comment](https://github.com/HumanCellAtlas/data-store/pull/2324#discussion_r317313026). I agree it is a bit confusing at first, but it does help reduce the amount of copypasta and number of functions that are mostly the same with just 2-3 lines different. 

I have moved this functionality to `params.py` (https://github.com/HumanCellAtlas/data-store/blob/dss-params/dss/operations/params.py#L137-L169) but I think we should keep the wrapper methods as-is.